src/plugins/intel_gna/gna_graph_compiler.cpp

```diff
@@ -952,6 +952,16 @@ void GNAGraphCompiler::ConcatPrimitive(InferenceEngine::CNNLayerPtr layer) {
         }
     }
 
+    // Concat axis validation
+    if (!GNALimitations::ValidateConvConcatAxis(concatLayer) && gnaFlags->enable_user_warning) {
+        std::ostringstream in_dims_oss;
+        auto in_dims = concatLayer->insData[0].lock()->getDims();
+        std::copy(in_dims.begin(), in_dims.end(), std::ostream_iterator<size_t>(in_dims_oss, ","));
+        std::cout << "Topology with layer: " + layer->name + ", type: " + layer->type +
```

Comment:
I think we need macro which can handle log level inside.

Reply:
I can think about it. The issue here is that gnaFlags is required and it's not available everywhere.

Reply:
I suggest we leave it for now as it is and rethink how to handle all the existing ways of printing debug/warning/error messages in one common way, maybe utilizing this new-for-gna config option. There are already defines/functions like gnalog, gnawarn, QUANTWARNING, as well as just fprintf used all over the code. It's worth a separate task to clean it all up, in my opinion. I wouldn't like to introduce yet another macro here.